crates/ruff_python_formatter/src/pattern/pattern_match_class.rs

```diff
@@ -109,46 +53,10 @@ impl NeedsParentheses for PatternMatchClass {
         //     (...)
         // ): ...
         // ```
-        let dangling = context.comments().dangling(self);
-        if !dangling.is_empty() {
-            if let Some(left_paren) = SimpleTokenizer::starts_at(self.cls.end(), context.source())
-                .find(|token| token.kind() == SimpleTokenKind::LParen)
-            {
-                if dangling
-                    .iter()
-                    .any(|comment| comment.start() < left_paren.start())
-                {
-                    return OptionalParentheses::Multiline;
-                };
-            }
-        }
-        OptionalParentheses::Never
-    }
```

Comment:
This specific logic is way simpler now -- we only care if there are dangling comments, not where they are, since these are dangling comments between `Point2D` and its arguments.